HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentReviewViewController.swift

```diff
@@ -211,6 +211,7 @@ public final class PaymentReviewViewController: UIViewController, UIGestureRecog
         infoBar.backgroundColor = UIColor.from(giniColor: giniHealthConfiguration.infoBarBackgroundColor)
         infoBarLabel.textColor = UIColor.from(giniColor: giniHealthConfiguration.infoBarTextColor)
         infoBarLabel.font = giniHealthConfiguration.customFont.regular
+        infoBarLabel.adjustsFontForContentSizeCategory = true
```

Comment:
[Clarification]: @ValentinaIancu-Gini why did we need to add it here? It was in the storyboard before 🤔?

Reply:
I've moved it from the storyboard to add it programmatically since the storyboard was complaining about it. 😵‍💫

Reply:
Cool thank you for the explanation